src/west/util.py

```diff
@@ -42,15 +42,20 @@ def west_topdir():
     Like west_dir(), but returns the path to the parent directory of the west/
     directory instead, where project repositories are stored
     '''
+    # If you change this function, make sure to update the bootstrap
+    # script's find_west_topdir().
+    def is_west_dir(d):
+        return os.path.isdir(d) and '.west_topdir' in os.listdir(d)
```

Comment:
`os.path.isfile(os.path.join(d, '.west_topdir'))` might be a bit more efficient.